## template_url_prepopulate_data.cc
# Adds DDG to the default search engines.
#
## util.cc
# Only snyc the prepoulated search engines when initializing the database.
# Prevents search engines from being overridden in future updates.
#
--- a/components/search_engines/template_url_prepopulate_data.cc
+++ b/components/search_engines/template_url_prepopulate_data.cc
@@ -59,6 +59,7 @@ constexpr EngineAndTier engines_default[
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
+    {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Note, the below entries are sorted by country code, not the name in comment.
--- a/components/search_engines/util.cc
+++ b/components/search_engines/util.cc
@@ -524,7 +524,7 @@ void GetSearchProvidersUsingLoadedEngine
 
   const int prepopulate_resource_keyword_version =
       TemplateURLPrepopulateData::GetDataVersion(prefs);
-  if (*resource_keyword_version < prepopulate_resource_keyword_version) {
+  if (*resource_keyword_version == 0) {
     MergeEnginesFromPrepopulateData(service, &prepopulated_urls, template_urls,
                                     default_search_provider,
                                     removed_keyword_guids);
